src/components/CreateSchema.tsx

```diff
@@ -1,4 +1,4 @@
-import {registerSchema } from "../services/chain/apis/extrinsic";
+import {createSchema } from "../services/chain/apis/extrinsic";
```

Comment:
Not wrong, but I'm surprised that linting/formatting didn't make sure it had a space. Might be worth checking to make sure linting is working.
```suggestion
import { createSchema } from "../services/chain/apis/extrinsic";
```

Reply:
Fixed the linting.  It was skipping .tsx files.